roles/docspell/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+#########################################################################
+# Title:            Sandbox: docspell                                   #
+# Author(s):        JigSawFr                                            #
+# URL:              https://github.com/saltyorg/Sandbox                 #
+# URL:              https://github.com/eikek/docspell                   #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: Check if db folder exists
+  ansible.builtin.stat:
+    path: "{{ server_appdata_path }}/{{ docspell_paths_folder }}/mariadb"
+  register: stat_docspell_db_folder
+
+- name: MariaDB Role
+  ansible.builtin.include_role:
+    name: mariadb
+  vars:
+    mariadb_name: "{{ docspell_name }}_mariadb"  # Remove when instance support merged into Saltbox
+    mariadb_instances: ["{{ docspell_name }}_mariadb"]
+    mariadb_paths_folder: "{{ docspell_name }}"
+    mariadb_paths_location: "{{ server_appdata_path }}/{{ docspell_paths_folder }}/mariadb"
+
+- name: Create docspell database
+  ansible.builtin.command: "docker exec {{ docspell_name }}_mariadb mysql -u root -p{{ mariadb_docker_envs_mysql_root_password }} -e 'create schema {{ docspell_name }} DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;'"
```

Comment:
Can we try using this Ansible module to create the db?

https://docs.ansible.com/ansible/latest/collections/community/mysql/mysql_db_module.html#ansible-collections-community-mysql-mysql-db-module